CMakeLists.txt

```diff
@@ -103,6 +103,13 @@ list(APPEND CMAKE_PREFIX_PATH /opt/rocm /opt/rocm/llvm /opt/rocm/hip /opt/rocm/h
 
 option(ENABLE_HIP_WORKAROUNDS Off)
 
+option(MIOPEN_USE_COMGR "When possible, use comgr to build kernels instead of offline tools" Off)
+if(MIOPEN_USE_COMGR)
```

Comment:
There's no point in checking `MIOPEN_USE_COMGR`, its already been defined.

Reply:
Oops))

Reply:
Done.